image_transport_py/README.md

```diff
@@ -0,0 +1,203 @@
+# image_transport_py: Python Bindings for ROS 2 Image Transport
```

Comment:
tutorial should point to https://github.com/ros-perception/image_transport_tutorials to avoid duplicate content